smart-contracts/Makefile

```diff
@@ -0,0 +1,24 @@
+# Makefile
+
+SHELL:=/bin/bash
+
+setup:
+	python3 -m venv venv
+	source ./venv/bin/activate
+	pip3 install -r requirements.txt
+	yarn
+
+# Run slither over entire directory
+# `make slither`
+slither: setup
+	slither . || true
+
+# Simple static analysis in a human-readable report over entire directory
+# `make slither-pretty-summary`
+slither-pretty-summary: setup
+	slither . --print human-summary
+
```

Comment:
Running any make command spits out this error:

`python3 -m venv venv
source ./venv/bin/activate
make: source: No such file or directory
make: *** [setup] Error 1`